swap/src/monero.rs

```diff
@@ -229,8 +229,8 @@ pub trait CreateWalletForOutput {
 }
 
 #[async_trait]
-pub trait CreateWalletForOutputThenLoadDefaultWallet {
-    async fn create_and_load_wallet_for_output_then_load_default_wallet(
+pub trait CreateWalletForOutputThenReloadWallet {
```

Comment:
I would just call this `create_from` and don't mention this reloading behaviour in the name. We simply guarantee to the caller that the current wallet remains loaded when you create a new one.